src/Hazelcast.Net/Core/RetryOptions.cs

```diff
@@ -12,12 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Hazelcast.Clustering;
-
 namespace Hazelcast.Core
 {
     /// <summary>
-    /// Represents a <see cref="RetryStrategy"/> configuration.
+    /// Represents the configuration for the retry strategy.
     /// </summary>
     public class RetryOptions
```

Comment:
It should be renamed as `ConnectionRetryOptions`

Reply:
ok